src/iblt.cpp

```diff
@@ -125,6 +153,17 @@ void CIblt::resize(size_t _expectedNumEntries)
     hashTable.resize(nEntries);
 }
 
+uint32_t CIblt::saltedHashValue(size_t hashFuncIdx, const std::vector<uint8_t> &kvec) const
+{
+    if (version > 0)
+    {
+        uint32_t seed = mapHashIdxSeeds.at(hashFuncIdx);
+        return MurmurHash3(seed, kvec) & KEYCHECK_MASK;
+    }
+    else
+        return MurmurHash3(hashFuncIdx, kvec);
```

Comment:
for consistency add & KEYCHECK_MASK here or remove it from the above (since its a nop because the return value is defined as a uint32_t).